R/snapshot.R

```diff
@@ -387,28 +381,66 @@ expect_snapshot_helper <- function(
   pass(NULL)
 }
 
-snapshot_accept_hint <- function(variant, file, reset_output = TRUE) {
+snapshot_hint <- function(
+  file,
+  name,
+  show_accept = TRUE,
+  reset_output = TRUE
+) {
+  if (in_check_reporter()) {
+    return("")
+  }
+
   if (reset_output) {
     local_reporter_output()
   }
 
-  if (is.null(variant) || variant == "_default") {
-    name <- file
+  id <- c(file, name)
```

Comment:
Q: The hint has changed from "Review the test file snapshots" to "Review this specific snapshot only", which causes work (delete the file name or many copy/paste's) to review more than one snapshot in a single test file. While the new code is accurate, it makes reviewing more time consuming.

I'd advocate for still returning the variant name for review. 

Expected:
* `testthat::snapshot_review("mytest/", "inst/apps/001-hello/tests/testthat")`
Not:
* `testthat::snapshot_review("mytest/002.png", "inst/apps/001-hello/tests/testthat")`

```suggestion
  id <- file
```

-------------------------

Or would it be possible to have the final comments suggest to review the test file's snaps.

Reply:
Ooooooh, I did wonder about that, but I thought that was the existing behaviour too. I removed the variant because it didn't work, but I think that's because it doesn't work when both file and variant are supplied.

(There's something about the code paths here that I find extremely confusing.)